docs/reference/announcements-release-notes/850/850-announcements.md

```diff
@@ -1,39 +1,36 @@
 ---
-id: announcements-850
+id: 850-announcements
 title: "8.5 - 8.3 Announcements"
-description: "Important announcements including deprecation & removal notices for the Camunda 8.5, 8.4, 8.3, and 8.2 releases."
+description: "Supported environment changes and breaking changes or deprecations for the Camunda 8.5, 8.4, and 8.3 releases."
+toc_max_heading_level: 2
 ---
 
-Important changes and updates for the Camunda 8.5, 8.4, and 8.3 releases are summarized below.
+Supported environment changes and breaking changes or deprecations for the Camunda 8.5, 8.4, and 8.3 releases are summarized below.
 
 ## Camunda 8.5
 
-| Release date | End of maintenance | Release notes                                        |
-| :----------- | :----------------- | :--------------------------------------------------- |
-| 9 April 2024 | 14 October 2025    | [8.5 release notes](/reference/release-notes/850.md) |
+| Release date | End of maintenance | Release notes                                                                        |
+| :----------- | :----------------- | :----------------------------------------------------------------------------------- |
+| 9 April 2024 | 14 October 2025    | [8.5 release notes](/reference/announcements-release-notes/850/850-release-notes.md) |
 
-### Updated SaaS URLs
+### Supported environment changes
 
-We will simplify the URL for Camunda 8 SaaS from cloud.camunda.io ([console.cloud.camunda.io](https://console.cloud.camunda.io/)) to camunda.io ([console.camunda.io](http://console.camunda.io/)).
+#### Zeebe Go client
 
-On or around July 9th, users will be directed to the new URLs. Both URLs will continue to be active for at least 18 months so navigation from supported versions of components like Operate is still possible.
-
-Internal allowlisting or active rules for [cloud.camunda.io](http://cloud.camunda.io/) must be transitioned to the new [camunda.io](http://camunda.io/) URL. This change primarily affects Console and Modeler. During sign up, users will be briefly redirected through [accounts.cloud.camunda.io](http://accounts.camunda.io/), which will also be updated.
+- Raised minimum Go version to 1.21 for the Zeebe Go client
 
-### Syntax changes in Helm chart
+#### Syntax changes in Helm chart
 
 A Camunda Helm chart upgrade is not possible from v9.x.x to v10.0.0 or v10.0.1. Instead, upgrade directly to v10.0.2+.
```

Comment:
Helm is noted in every version 8.3-8.7, but not yet in 8.8?

Reply:
Create an issue for this so Immi and Cole can follow up. I assume there will be changes, although I don't know if I'd call them syntax changes. They do have configuration changes version to version that we probably want to elevate to this page (see https://docs.camunda.io/docs/8.7/self-managed/setup/upgrade/?upgrades=8.6#update-your-configuration).

In your issue, you can make a recommendation, but I don't see this needing to be _solved_ within this PR.